src/Components/CreateImageWizardV2/validators.ts

```diff
@@ -36,3 +38,24 @@ export const isBlueprintNameValid = (blueprintName: string) =>
 export const isBlueprintDescriptionValid = (blueprintDescription: string) => {
   return blueprintDescription.length <= 250;
 };
+export const isFileSystemConfigValid = (partition: Partition[]) => {
+  const duplicates = getDuplicateMountPoints(partition);
+  return !(duplicates?.length !== 0);
```

Comment:
This is a bit confusing, what if we just did `duplicates.length === 0 ? true : false`?